src/pages/mathematics/factorials/index.md

```diff
@@ -3,13 +3,28 @@ title: Factorials
 ---
 ## Factorials
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/mathematics/factorials/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Factorials are the product of a positive integer and all the positive integers below it. They're denoted by an exclamation point, like this: `n!`. Factorials can be represented iteratively as:
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+`n! = n * n-1 * n-2 * ... * 2 * 1`
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+and recursively as:
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+`n! = n * (n-1)!`
+
+###Examples
```

Comment:
@rosenshine Please correct the formatting of the sub headers. The i will merge your changes.